web/src/context/Stats19.svelte

```diff
@@ -0,0 +1,234 @@
+      "all",
+      [">=", ["get", "year"], state.minYear],
+      ["<=", ["get", "year"], state.maxYear],
+      includeTypes,
+    ];
+  }
+
+  function casualtyTypes(props: { [name: string]: any }): string {
+    let list = [];
+    if (props.pedestrian) {
+      list.push("pedestrian");
+    }
+    if (props.cyclist) {
+      list.push("cyclist");
+    }
+    return list.join(", ");
+  }
+
+  function onClick(e: CustomEvent<LayerClickInfo>) {
+    window.open(
+      `https://www.cyclestreets.net/collisions/reports/${
+        e.detail.features[0].properties!.accident_index
+      }`,
+      "_blank",
+    );
+  }
+
+  // From
+  // https://data.dft.gov.uk/road-accidents-safety-data/dft-road-casualty-statistics-road-safety-open-dataset-data-guide-2023.xlsx.
+  // The "not a pedestrian", "data missing", and "unknown" cases are filtered out
+  // upstream.
+  let pedestrianLocation: { [name: number]: string } = {
+    1: "Crossing on pedestrian crossing facility",
+    2: "Crossing in zig-zag approach lines",
+    3: "Crossing in zig-zag exit lines",
+    4: "Crossing elsewhere within 50m. of pedestrian crossing",
+    5: "In carriageway, crossing elsewhere",
+    6: "On footway or verge",
+    7: "On refuge, central island or central reservation",
+    8: "In centre of carriageway - not on refuge, island or central reservation",
+    9: "In carriageway, not crossing",
+  };
+  let pedestrianMovement: { [name: number]: string } = {
+    1: "Crossing from driver's nearside",
+    2: "Crossing from nearside - masked by parked or stationary vehicle",
+    3: "Crossing from driver's offside",
+    4: "Crossing from offside - masked by parked or stationary vehicle",
+    5: "In carriageway, stationary - not crossing (standing or playing)",
+    6: "In carriageway, stationary - not crossing (standing or playing - masked by parked or stationary vehicle)",
+    7: "Walking along in carriageway, facing traffic",
+    8: "Walking along in carriageway, back to traffic",
+  };
+  let severity: { [name: number]: string } = {
+    1: "Fatal",
+    2: "Serious",
+    3: "Slight",
+  };
+
+  let fatalColor = "#080C54";
+  let seriousColor = "#1F9EB7";
+  let slightColor = "#CDE594";
+  let severityLegend = {
+    Fatal: fatalColor,
+    Serious: seriousColor,
+    Slight: slightColor,
+  };
+</script>
+
+<button class="secondary" on:click={() => (show = !show)}>Stats19</button>
+{#if show}
+  <HelpButton>
+    <p>
+      This layer shows collisions recorded in the <a
+        href="https://www.data.gov.uk/dataset/cb7ae6f0-4be6-4935-9277-47e5ce24a11f/road-safety-data"
+        target="_blank"
+      >
+        DfT stats19
+      </a>
+      dataset, as of 30 September 2024. Please note these limitations:
+    </p>
+    <ul>
+      <li>Only collisions between 2017 and 2023 are included</li>
+      <li>
+        This tool is intended to be used at high-zoom levels, while inspecting a
+        scheme or development area. Not all points are shown at lower zoom
```

Comment:
Take it or leave it copy nit: Do people know which zoom levels are high vs low? 

Personally I think "zoomed out" and "zoomed in" are more intuitive, but are kind of vague in their own way.

Reply:
Done